commit b739759c4e96060b7882c22364a4e5d5b3fc7458
Author: Ravi Bangoria <ravi.bangoria@amd.com>
Date:   Thu Jun 20 05:41:04 2024 +0000

    perf doc: Add AMD IBS usage document
    
    Add a perf man page document that describes how to exploit AMD IBS with
    Linux perf. Brief intro about IBS and simple one-liner examples will help
    naive users to get started. This is not meant to be an exhaustive IBS
    guide. User should refer latest AMD64 Architecture Programmer's Manual
    for detailed description of IBS.
    
    Usage:
    
      $ man perf-amd-ibs
    
    Signed-off-by: Ravi Bangoria <ravi.bangoria@amd.com>
    Reviewed-by: Arnaldo Carvalho de Melo <acme@redhat.com>
    Cc: ananth.narayan@amd.com
    Cc: sandipan.das@amd.com
    Cc: santosh.shukla@amd.com
    Signed-off-by: Namhyung Kim <namhyung@kernel.org>
    Link: https://lore.kernel.org/r/20240620054104.815-1-ravi.bangoria@amd.com

diff --git a/tools/perf/Documentation/perf.txt b/tools/perf/Documentation/perf.txt
index 09f516f3fdfb..cbcc2e4d557e 100644
--- a/tools/perf/Documentation/perf.txt
+++ b/tools/perf/Documentation/perf.txt
@@ -82,7 +82,8 @@ linkperf:perf-stat[1], linkperf:perf-top[1],
 linkperf:perf-record[1], linkperf:perf-report[1],
 linkperf:perf-list[1]
 
-linkperf:perf-annotate[1],linkperf:perf-archive[1],linkperf:perf-arm-spe[1],
+linkperf:perf-amd-ibs[1], linkperf:perf-annotate[1],
+linkperf:perf-archive[1], linkperf:perf-arm-spe[1],
 linkperf:perf-bench[1], linkperf:perf-buildid-cache[1],
 linkperf:perf-buildid-list[1], linkperf:perf-c2c[1],
 linkperf:perf-config[1], linkperf:perf-data[1], linkperf:perf-diff[1],